package/src/index.ts

```diff
@@ -39,16 +39,18 @@ import type {
   MarketTradeAssetRef,
   TradeAssetRef as InternalTradeAssetRef,
   LinkedAccountStatusRef,
+  Coordinates,
+  Address,
 } from "./specs/RNCandle.nitro";
 
 export class CandleClient {
   private candle: RNCandle;
 
-  constructor(appUser: AppUser) {
+  constructor(appUser: AppUser, accessGroup: string | undefined = undefined) {
```

Comment:
```suggestion
  constructor(appUser: AppUser, accessGroup?: string) {
```
#pp  simpler and stricter